cpp/src/traversal/bfs.cu

```diff
@@ -487,7 +488,7 @@ void bfs(raft::handle_t const &handle,
   static_assert(std::is_floating_point<WT>::value,
                 "Unsupported edge weight type. Use floating point types");  // actually, this is
                                                                             // unnecessary for BFS
-  if (handle.comms_initialized()) {
+  if (handle.comms_initialized() && !mg_batch) {
     CUGRAPH_EXPECTS(sp_counters == nullptr,
                     "BFS Traversal shortest path is not supported in OPG path");
```

Comment:
We are asked to avoid using OPG naming.

Reply:
Should this be done in this PR for the `bfs` code or should this be update in an unifying PR?